ui/partials/perf/comparectrl.html

```diff
@@ -10,8 +10,14 @@
     <div id="subtest-summary" ng-if="!dataLoading && !errors.length">
       <h1>Perfherder Compare Revisions</h1>
       <ul class="list-inline push-information">
-        <li><strong>Base</strong> - <a href="{{originalProject.getRevisionHref(originalRevision)}}">{{originalRevision}}</a> ({{originalProject.name}})</li>
-        <li><strong>New</strong> - <a href="{{newProject.getRevisionHref(newRevision)}}">{{newRevision}}</a> ({{newProject.name}})</li>
+        <li>
+          <strong>Base</strong> -
+          <revision-describe project="originalProject" project-revision="originalRevision" author="originalAuthor" comment="originalComments"></revision-describe>
+        </li>
+        <li>
+          <strong>New</strong> -
+          <revision-describe project="newProject" project-revision="newRevision" author="newAuthor" comment="newComments"></revision-describe>
+        </li>
       </ul>
```

Comment:
So in fact I think the directive should cover *everything* between lines 14 and 21, since this is currently duplicated. That is, it should have 8 parameters:

original-project, original-revision, original-author, original-comments, new-project, new-revision, new-author, new-comments